include/psa/crypto_compat.h

```diff
@@ -50,6 +50,8 @@ extern "C" {
 typedef MBEDTLS_PSA_DEPRECATED size_t mbedtls_deprecated_size_t;
```

Comment:
Minor: typo in commit 27f8944 - `Remove the values of curve encodings that is based` - encodings/values are plural, so I'd assume that it should be that **are** based.
Also, `andinclude` -> `and include`

Reply:
I fixed the commit message.